src/main/java/uk/gov/hmcts/juror/api/moj/report/bespoke/JurorAmendmentByPoolReport.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.juror.api.moj.report.bespoke;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.JurorPool;
+import uk.gov.hmcts.juror.api.moj.domain.PoolRequest;
+import uk.gov.hmcts.juror.api.moj.domain.Role;
+import uk.gov.hmcts.juror.api.moj.report.AbstractReport;
+import uk.gov.hmcts.juror.api.moj.service.JurorPoolService;
+import uk.gov.hmcts.juror.api.moj.service.JurorServiceMod;
+import uk.gov.hmcts.juror.api.moj.service.UserService;
+import uk.gov.hmcts.juror.api.moj.service.audit.JurorAuditService;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+public class JurorAmendmentByPoolReport extends AbstractJurorAmendmentReport {
+
+    private final JurorPoolService jurorPoolService;
+
+    @Autowired
+    public JurorAmendmentByPoolReport(
+        JurorPoolService jurorPoolService,
+        JurorAuditService jurorAuditService,
+        JurorServiceMod jurorService,
+        UserService userService) {
+        super(jurorAuditService, jurorService, userService);
+        this.jurorPoolService = jurorPoolService;
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public JurorAmendmentReportResponse getStandardReportResponse(
+        StandardReportRequest request) {
+        JurorAmendmentReportResponse response = new JurorAmendmentReportResponse();
+
```

Comment:
As above, could add some logging at start and end of this method

Reply:
See Previous Comment